code/lib/core-common/src/presets.ts

```diff
@@ -210,8 +209,7 @@ async function getContent(input: any) {
     return rest;
   }
   const name = input.name ? input.name : input;
-
-  return interopRequireDefault(name);
+  return import(name);
```

Comment:
@ndelangen this is a bit of a risky change, and needs your 👀.

I do know that without this, it's going to be very difficult to use vitest, since it only mocks `import` and not `require`

Reply:
I've managed to revert this in https://github.com/storybookjs/storybook/pull/24415/commits/6dfcad5214340af3080d958777599aaa5efed612 and use `mock-require` to mock the presets instead. Seems to work